Tasks/DeployAzureResourceGroup/Utility.ps1

```diff
@@ -18,7 +18,8 @@ function Get-SingleFile($files, $pattern)
 function Get-File($pattern)
 {
     #Find the File based on pattern
-    
+    import-module "Microsoft.TeamFoundation.DistributedTask.Task.Common"
```

Comment:
Why are we importing dlls inside the helper funtions of the task. If we import it once in the main script, i.e. DeployAzureResourceGroup in this case would not that be sufficient?